index.bs

```diff
@@ -260,6 +265,9 @@ interface ImageCapture {
 
   <dt><dfn dict-member for="PhotoSettings"><code>fillLightMode</code></dfn></dt>
   <dd>This reflects the desired <a>fill light mode</a> (flash) setting.</dd>
+
+  <dt><dfn dict-member for="PhotoSettings"><code>imageFormat</code></dfn></dt>
+  <dd>This reflects the desired <a>image format</a> for the captured image. If the UA supports desired <a>image format</a>, the {{Blob/type!!attribute}} of the {{Blob}} produced by <a>takePhoto()</a> MUST be one of the {{PhotoCapabilities/imageFormat}} types.</dd>
```

Comment:
I'd phrase the second sentence something like: `When the UA returns a {{Blob}} from {{takePhoto()}}, the format and {{Blob/type}} MUST be one of the supported <a>image formats</a> enumerated in {{PhotoCapabilities/imageFormat}}, if any.`